components/Payment/PaymentList/PaymentList.scss

```diff
@@ -0,0 +1,66 @@
+.payment-list {
+  display: flex;
+  flex-flow: column;
+  row-gap: 48px;
+
+  > *:not(:last-child) {
+    border-bottom: 1px solid rgba($color-white-alpha-300, 0.3);
+  }
+
+  &__item {
+    padding-bottom: 48px;
+
+    @include tablet-xl {
+      display: flex;
+      gap: 24px;
+    }
+
+    @include desktop-xl {
+      gap: 32px;
+    }
+  }
+
+  &__title {
+    margin-bottom: 24px;
+    max-width: 230px;
+
+    @include tablet-xl {
+      padding-right: 24px;
+      min-width: 306px;
+    }
+
+    @include desktop {
+      min-width: 365px;
+    }
+
+    @include desktop-xl {
+      min-width: 440px;
+    }
+  }
+
+  &__inner {
+    display: flex;
+    gap: 16px;
+
+    > ul {
+      width: calc(50% - 16px);
+    }
```

Comment:
I don't think using the tag is the right thing to do

It would be better to use an additional class